templates/default/event/eventlistboxrow.html

```diff
@@ -19,12 +19,23 @@
             <br>{$event.date|date_format:"%x"}
         {/if}
         {$imadded=0}
-        {foreach $event.userlist as $user}
-		{if $layout.logid == $user.id}
-			{$imadded=1}
-		{/if}
-		<br><a href="?m=userinfo&amp;id={$user.id}" style="font-weight: normal">{$user.name|truncate:"25":"...":true}</a>
-        {/foreach}
+	{$user_row_limit = intval(ConfigHelper::getConfig('phpui.timetable_user_row_limit', 4))}
```

Comment:
Nadal coś z formatowaniem. Wystarczy patrzeć jak jest w pliku i dostosowywać się do tego.

Reply:
Tą akurat linijkę dodałeś Ty. Chyba że masz na myśli jakiś inny fragment już poprawiam to.

Reply:
Poprawiłem.